app/src/main/java/org/wikipedia/watchlist/WatchListItem.kt

```diff
@@ -0,0 +1,18 @@
+package org.wikipedia.watchlist
+
+import com.google.gson.annotations.SerializedName
+
+class WatchListItem {
```

Comment:
Some of the fields in this class are awfully similar to `MwQueryPage`.  Could this be a subclass?

Reply:
yes, it is actually a list of MwQueryPage(s)... so will use that itself.